manifest.json

```diff
@@ -59,6 +59,12 @@
       "all_frames": false,
       "js": ["content/install-hook-userstyles.js"]
     },
+    {
+      "matches": ["https://openusercss.org/*", "https://openusercss.com/*"],
+      "run_at": "document_start",
+      "all_frames": false,
+      "js": ["content/openusercss-integration.js"]
```

Comment:
Other content scripts use a different naming scheme so this one should be named as `install-hook-openusercss.js`.

Reply:
While the filename doesn't hold much meaning in terms of function, I named it like this because there isn't any install hook in it yet, only the handshake. If you insist, I don't mind renaming it at all.

Reply:
Well, the primary use case will be installation of styles so it's better to have the name follow the scheme.

Reply:
That's a good point, it will have that down the line, I renamed it :+1: